azurerm/internal/services/keyvault/key_vault_certificate_resource.go

```diff
@@ -117,6 +117,11 @@ func resourceArmKeyVaultCertificate() *schema.Resource {
 										Required: true,
 										ForceNew: true,
 									},
+
+									"certificate_type": {
+										Type:     schema.TypeString,
+										Optional: true,
```

Comment:
what's the supported possible values for this field? can we add some tests for these values?